### ISSUE TITLE
Change correct indication on answer options

### ISSUE THREAD

**Issue by etenoch** [qlicker/qlicker#140]
Use something in addition to colors to indicate the correctness of an answer. Green/red is not accessibility friendly and does not work on image answer options.

<img width="427" alt="screen shot 2017-04-05 at 2 28 11 pm" src="https://cloud.githubusercontent.com/assets/1919500/24720779/3dadc41e-1a0c-11e7-8321-0cf82908e21b.png">
